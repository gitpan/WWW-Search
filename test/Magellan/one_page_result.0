<HTML>
<HEAD><TITLE>Search Results</TITLE></HEAD>
<BODY BGCOLOR="#ffffff">

<CENTER>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
 <TR>
<TD ALIGN=LEFT VALIGN=top ROWSPAN=4>
<A HREF="http://www.mckinley.com/">
<IMG SRC="/img/magellan/mglobe.gif" BORDER=0 WIDTH=49 HEIGHT=41 ALIGN=left></A>
</TD>
<TD ALIGN=middle VALIGN=top ROWSPAN=4>
<A HREF="http://www.mckinley.com/"><IMG SRC="/img/magellan/magellan.gif" BORDER=0 WIDTH=224 HEIGHT=41 ALIGN=middle ALT="Magellan Internet Guide"></A><BR>
<IMG SRC="/img/magellan/white.gif" HEIGHT=4 WIDTH=1><BR>
<FONT FACE="arial,helvetica" SIZE="-2">
<A HREF="http://www.mckinley.com/">HOME</A> | <A HREF="/magellan/Reviews/index.magellan.html">REVIEWS</A></FONT>
</TD><TD WIDTH=4 ROWSPAN=4></TD>
<TD HEIGHT="2" COLSPAN=3 NOWRAP></TD></TR>

<TR>


<FORM ACTION="/search.gw" METHOD=get>
<TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
<INPUT NAME=search VALUE="disestablishmentarianism" SIZE=30>
<INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
</TD></TR>
<TR>
 <TD ALIGN=left NOWRAP>
 <INPUT TYPE=radio NAME=c VALUE=web.guide
    > <FONT SIZE="-1">Reviewed Sites Only</FONT></TD>
 <TD WIDTH=3 ROWSPAN=2></TD>
 <TD ALIGN=left>
  <INPUT TYPE=radio NAME=c VALUE=web.green
        ><FONT SIZE="-1" COLOR=#008C05>Green Light</FONT><FONT SIZE="-1"> sites only</FONT>
 </TD>
</TR>
<TR>
 <TD ALIGN=left><INPUT TYPE=radio NAME=c VALUE=web
   CHECKED  > <FONT SIZE="-1">The Entire Web</FONT>
<INPUT TYPE=hidden NAME=look VALUE=magellan>
</TD>

<!-- End of search and collection section -->


<TD ALIGN=left><FONT SIZE="-1"><A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A></FONT></TD></TR></TABLE>

<input type=hidden name=sorig value=rpage>
</FORM>
</CENTER>
<P>


 <CENTER><!-- Ad Start -->
<!-- Ad param svcpage=/search.gw keyword=disestablishmentarianism svcname=msr ip=128.9.160.184 user_agent=WWW::Search/1.015 -->
<a href="/relocate/ad=33377_icn0317_coolsavings_banner100.gif|pk=336878;http://www.coolsavings.com/index.asp?Ref=exct~token=gen9" onMouseOver="window.status='Click Here';return true" language="Javascript"><img src="/img/ads/33377_icn0317_coolsavings_banner100.gif" ALT="Click Here" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here </B></FONT></CENTER></a>
<!-- Ad Stop -->
</CENTER>

<BLOCKQUOTE>

<B>8</B> results returned, ranked by relevance.

<!-- IF statement for Green Light text -->



<FORM ACTION="/search.gw" METHOD=get>
    <INPUT TYPE=hidden NAME=search VALUE="disestablishmentarianism">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
                <INPUT TYPE=hidden NAME=start VALUE=0>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
            
<!--search results-->

   <HR>
  <B><A HREF="http://www.flash.net/~leenhout/aboutme.htm">About Me</A></B>&nbsp;&nbsp;&nbsp;
  <B>54%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1099193&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
       I like Shakespeare, Sondheim musicals, sci-fi novels (although lately it's been Ayn Rand), and movies, TV, etc.  Princess Bride is a classic movie, Star Trek is cool in all its forms, and My So-Called Life should never have been taken off the air.
    <BR>
  <I> <A HREF="http://www.flash.net/~leenhout/aboutme.htm">http://www.flash.net/~leenhout/aboutme.htm</A></I>
  <BR>
   <HR>
  <B><A HREF="http://sex.is.my.addiction.com/">sex.web disclaimer</A></B>&nbsp;&nbsp;&nbsp;
  <B>51%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=326621&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
       If you click yes below you: are guaranteeing your age to us, and that you are aware that this site contains links to sexually explicit material and you may view, read or hear material that is considered offensive by some people. web administrators not liable and agree to indemnify them and hold them harmless from any losses, liability, cost or expense incurred as a result of any information,.
    <BR>
  <I> <A HREF="http://sex.is.my.addiction.com/">http://sex.is.my.addiction.com/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.hotwired.com/synapse/feature/97/29/biggs0a_text.html">HotWired: Synapse - Feature</A></B>&nbsp;&nbsp;&nbsp;
  <B>45%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1337916&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
       Boiled down to the basics, libertarianism is based on the assumption that no government is good government.  Free markets are good, personal liberties are good, self-determination is good.
    <BR>
  <I> <A HREF="http://www.hotwired.com/synapse/feature/97/29/biggs0a_text.html">http://www.hotwired.com/synapse/feature/97/29/biggs0a_text.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.algonet.se/~spasm/">MR.FISK. NOTHiNG NET</A></B>&nbsp;&nbsp;&nbsp;
  <B>44%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=426719&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
      MR.FISK. NOTHiNG NET.
    <BR>
  <I> <A HREF="http://www.algonet.se/~spasm/">http://www.algonet.se/~spasm/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.geocities.com/RodeoDrive/3599/">Warning for Darren's Homepage</A></B>&nbsp;&nbsp;&nbsp;
  <B>40%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=1618209&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
        blow job boobies boobs breast bunny butt cybersex dominant erotica females girls hardcore head hooters hot hustler jugs labia lips live lust masturbate model movies mpeg naked nipples nudes oral penetration penthouse phonesex pictures playboy porn pornography quicktime sex.
    <BR>
  <I> <A HREF="http://www.geocities.com/RodeoDrive/3599/">http://www.geocities.com/RodeoDrive/3599/</A></I>
  <BR>
   <HR>
  <B><A HREF="http://members.tripod.com/~tritemp/index.htm">Triumvirate</A></B>&nbsp;&nbsp;&nbsp;
  <B>39%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=408413&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
       I as a young man with long wavey black hair shoulder length and with pail skin.  I carried a weapon, a bow, not just any weapon the weapon of a young wizard.
    <BR>
  <I> <A HREF="http://members.tripod.com/~tritemp/index.htm">http://members.tripod.com/~tritemp/index.htm</A></I>
  <BR>
   <HR>
  <B><A HREF="http://capo.org/premise/96/mar/p960304.html">From Reformation to Revolution: 1500-1650 - Premise - Vol
III, No 3, 1...</A></B>&nbsp;&nbsp;&nbsp;
  <B>01%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=122287&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
        Similar to Calvin, Luther believed that God had ordained both church and state as separate, but legitimate, spheres. &quot; [19] According to Thielicke, &quot;Luther pointed out again and again that the state must not concern itself with man's salvation.
    <BR>
  <I> <A HREF="http://capo.org/premise/96/mar/p960304.html">http://capo.org/premise/96/mar/p960304.html</A></I>
  <BR>
   <HR>
  <B><A HREF="http://www.angelfire.com/de/exposition/cyberart.html">This is a piece of cyber art!</A></B>&nbsp;&nbsp;&nbsp;
  <B>01%</B>&nbsp;&nbsp;&nbsp;
  <A HREF="/search.gw?lk=magellan&c=web&qbe=235781&ctag=35197E30&showSummary=true"><font size="-1">[find similar]</A></FONT><BR>
       Porno buck sex chat scream, sex chat I hear xxx nude through your bones close to.  Frantically video suck, hoping that this will finish me off.
    <BR>
  <I> <A HREF="http://www.angelfire.com/de/exposition/cyberart.html">http://www.angelfire.com/de/exposition/cyberart.html</A></I>
  <BR>
 <HR>
</FORM>
<FORM ACTION="/search.gw" METHOD=get>
             <INPUT TYPE=hidden NAME=search VALUE="disestablishmentarianism">
    <INPUT TYPE=hidden NAME=look VALUE=magellan>
       <INPUT TYPE=hidden NAME=start VALUE=0>
    <INPUT TYPE=hidden NAME=collection VALUE=web>
          </FORM>

Try your search on:&nbsp;
  <A HREF="http://www.excite.com/search.gw?s=disestablishmentarianism"><B>Excite</B></A>
    or <A HREF="http://webcrawler.com/cgi-bin/WebQuery?text=disestablishmentarianism&src=exciterecip"><B>Webcrawler</B></A><P>
   
</BLOCKQUOTE>
<CENTER>

<P><CENTER><!-- Ad Start -->
<a href="/relocate/ad=33377_icn0317_coolsavings_banner100.gif|pk=336878;http://www.coolsavings.com/index.asp?Ref=exct~token=gen9" onMouseOver="window.status='Click Here';return true" language="Javascript"><img src="/img/ads/33377_icn0317_coolsavings_banner100.gif" ALT="Click Here" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here </B></FONT></CENTER></a>
<!-- Ad Stop -->
</CENTER><P>

<FORM ACTION="/search.gw" METHOD=get>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=2>
<TR>


<TD ALIGN=middle VALIGN=middle WIDTH=120 ROWSPAN=3>
<TABLE BORDER=1 CELLPADDING=2><TR><TD ALIGN=middle VALIGN=middle HEIGHT=60
WIDTH=90 BGCOLOR="FFFFCC" NOWRAP><A HREF="/magellan/bookmark.html"><FONT SIZE="-1">Bookmark<BR>
Magellan<BR></FONT>NOW!<BR>
<FONT SIZE="-2">click here for details</FONT></A></TD></TR></TABLE></TD>


<TD ALIGN=left VALIGN=top COLSPAN=3 NOWRAP>
<INPUT NAME=search VALUE="disestablishmentarianism" SIZE=30>
<INPUT TYPE=IMAGE SRC="/img/magellan/searchbut.gif" BORDER=0 ALIGN="top" VALUE="Search Magellan" WIDTH=67 HEIGHT=30>
</TD></TR>
<TR>
 <TD ALIGN=left>
 <INPUT TYPE=radio NAME=c VALUE=web.guide
    > <FONT SIZE="-1">Reviewed Sites Only</FONT></TD>
 <TD WIDTH=3 ROWSPAN=2></TD>
 <TD ALIGN=left>
  <INPUT TYPE=radio NAME=c VALUE=web.green
        ><FONT SIZE="-1" COLOR=#008C05>Green Light</FONT><FONT SIZE="-1"> sites only</FONT>
 </TD>
</TR>
<TR>
 <TD ALIGN=left><INPUT TYPE=radio NAME=c VALUE=web
   CHECKED  > <FONT SIZE="-1">The Entire Web</FONT>
 <INPUT TYPE=hidden NAME=look VALUE=magellan>
 </TD>

 <TD ALIGN=left><FONT SIZE=-1>
 <A HREF="/magellan/Info/advancedtips.html">Search Tips and Help</A></FONT></TD></TR>
 </TABLE>
<input type=hidden name=sorig value=rpage>
</FORM>


<P><BR>
<CENTER>
<FONT FACE="arial,helvetica" SIZE="-1"><A HREF="http://www.mckinley.com/">Home</A> | <A
HREF="/magellan/Reviews/index.magellan.html">Reviews</A> | <A HREF="/magellan/Info/advancedtips.html">Help</A> | <A HREF="/magellan/Info/addsite.html">Add Site</A><P></FONT>
</FORM>
<P>
<HR>
<FONT SIZE=-1>
Explore <A HREF=/magellan/advertising.html>advertising</A> opportunities with Magellan.<BR>
Copyright &copy;1997 The McKinley Group, Inc., a subsidiary of <A HREF="http://www.excite.com/?Mh-a-t">Excite, Inc.</A>.  All rights reserved.
</FONT>
</CENTER>
<IMG SRC="http://ad.preferences.com/ping.ng;spacedesc=TCExciteData_Excite_1x1_RunOfSite_Any&event=KeywordResults" HEIGHT=1 WIDTH=1 BORDER=0>
</BODY>
</HTML>







