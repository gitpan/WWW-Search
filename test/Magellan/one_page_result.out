http://www.flash.net/~leenhout/aboutme.htm
http://sex.is.my.addiction.com/
http://www.hotwired.com/synapse/feature/97/29/biggs0a_text.html
http://www.algonet.se/~spasm/
http://www.geocities.com/RodeoDrive/3599/
http://members.tripod.com/~tritemp/index.htm
http://capo.org/premise/96/mar/p960304.html
http://www.angelfire.com/de/exposition/cyberart.html
